themes/becker.omp.json

```diff
@@ -0,0 +1,100 @@
+  "blocks": [
+    {
+      "alignment": "left",
+      "segments": [
+        {
+          "background": "#333333",
+          "foreground": "#78fb26",
+          "leading_diamond": "\ue0b6",
+          "properties": {
+            "template": "@{{ .UserName }} "
+          },
+          "style": "diamond",
+          "trailing_diamond": "\ue0b0",
+          "type": "session"
+        },
+        {
+          "background": "#1BD760",
+          "foreground": "#434343",
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "template": " \uf9c6 {{ .Icon }}{{ if ne .Status \"stopped\" }}{{ .Artist }} - {{ .Track }}{{ end }} "
+          },
+          "style": "powerline",
+          "type": "spotify"
+        },
+        {
+          "background": "#ff0000",
+          "foreground": "#000000",
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "style": "folder",
+            "template": " \ue5ff {{ .Path }} "
+          },
+          "style": "powerline",
+          "type": "path"
+        },
+        {
+          "background": "#fffb38",
+          "foreground": "#193549",
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "fetch_stash_count": true,
+            "fetch_upstream_icon": true,
+            "template": " {{ .UpstreamIcon }}{{ .HEAD }}{{ if gt .StashCount 0 }} \uf692 {{ .StashCount }}{{ end }} "
+          },
+          "style": "powerline",
+          "type": "git"
+        },
+        {
+          "background": "#6CA35E",
+          "foreground": "#ffffff",
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "template": " \ue718 {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }} "
+          },
+          "style": "powerline",
+          "type": "node"
+        },
+        {
+          "background": "#96df3d",
+          "foreground": "#000000",
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "template": " \ufcb5 {{ .Name }} "
+          },
+          "style": "powerline",
+          "type": "shell"
+        },
+        {
+          "background": "#000000",
+          "foreground": "#ea0404",
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "template": " \uf0e7 "
+          },
+          "style": "powerline",
+          "type": "root"
+        },
+        {
+          "background": "#fc0000",
+          "background_templates": [
+            "{{ if gt .Code 0 }}#f1184c{{ end }}"
+          ],
+          "foreground": "#ffffff",
+          "properties": {
+            "always_enabled": true,
+            "template": "<transparent>\ue0b0</> \uf1b6 <background,transparent>\ue0b4</>"
+          },
+          "style": "plain",
+          "type": "exit"
+        }
+      ],
+      "type": "prompt"
+    }
+  ],
+  "final_space": true,
+  "version": 1
```

Comment:
@RobertoASF the config is at version 2 right now, can you update oh-my-posh so you can also auto-migrate this theme?

Reply:
ok i do it now

Reply:
It seems to not have been migrated so I'll have a look for you on this one.